bin/helm-generate-raw-manifest-cluster-prep.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eo pipefail
+
+cd "$(git rev-parse --show-toplevel)"
+
+pushd helm/conjur-config-cluster-prep > /dev/null
+    mkdir -p generated
+
+    helm template cluster-prep . \
+        -f sample-values.yaml \
+        --render-subchart-notes \
+        --skip-tests > generated/conjur-config-cluster-prep.yaml
+
+    yq eval -i \
```

Comment:
Cool!!! This is much better than archaic sed/awk.

Reply:
Maybe as a followup to this (certainly not for this PR), maybe we can use `yq` to add angled brackets around the "fill-in-your-values" fields in the generated templates?

Or is that more trouble than it's worth?

Reply:
Hmm yeah that would work - although then wouldn't it be a bit confusing that we have 2 sources of sample values: 1 for rendering via helm template (`sample-values.yaml`) and another from somewhere else just to put the values in angled brackets?